pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "joladnijo-backend"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
Itt szerintem jó lenne valami másnak lennie (ha annyira nem is kritikus)

Reply:
Lehet, hogy nem is lenne szukseg az `authors` propertyre. Kitorlom ezt a sort.

Reply:
Elvileg kötelező mező (python-poetry.org/docs/pyprojetct/#authors)

Reply:
Akkor meg hagyom igy. Ha akarjatok, akkor hozza lehet adni a sajat neveteket.